internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -263,7 +263,7 @@ func (rp *RegistryPod) podForBundleRegistry() (*corev1.Pod, error) {
 // getContainerCmd uses templating to construct the container command
 // and throws error if unable to parse and execute the container command
 func (rp *RegistryPod) getContainerCmd() (string, error) {
-	const containerCommand = "/bin/mkdir -p {{ .DBPath }} &&" +
+	const containerCommand = "/bin/mkdir -p /database &&" +
```

Comment:
Does the template not replace `.DBPath` with `/database`?

Reply:
I think we may need to do `filepath.Dir(rp.DBPath)` and use that here.